app/src/main/java/com/example/memeitupapp/ui/meme_it_up/MemeItUpActivity.kt

```diff
@@ -1,12 +1,13 @@
-package com.example.memeitupapp
+package com.example.memeitupapp.ui.meme_it_up
```

Comment:
do not use **_** to separate words in packages, use **.**

Reply:
done, thanks